pidfile/pidfile.go

```diff
@@ -0,0 +1,30 @@
+package pidfile
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+// WritePid : write pid to given file
+func WritePid(pidfile string) error {
+	dir, filename := filepath.Split(pidfile)
+	tmpfile, err := ioutil.TempFile(dir, filename+".*")
+	if err != nil {
+		return errors.Wrap(err, "Cloud not create tempfile")
+	}
+	_, err = tmpfile.WriteString(fmt.Sprintf("%d", os.Getpid()))
+	if err != nil {
+		tmpfile.Close()
+		return errors.Wrap(err, "Cloud not write pid to tempfile")
+	}
+	tmpfile.Close()
+	err = os.Rename(tmpfile.Name(), pidfile)
+	if err != nil {
+		return errors.Wrap(err, "Cloud not rename pidfile")
```

Comment:
`s/Cloud/Could/`

Reply:
In addition, this block has same problem as below.

https://github.com/kazeburo/chocon/pull/33/files#r259357109